api/db/file.js

```diff
@@ -1,3 +1,5 @@
+const o = v => (typeof v === 'object' ? v : JSON.parse(v));
+
```

Comment:
Bookshelf is inconsistent with JSON columns.  When you assign to them, you have to assign them as strings.  When you fetch them from the database, though, they come back as objects.  If you read the property from the model object after saving it, but without fetching it again, then it's a string.  This just tries to normalize for that.